20210325/MINUTES.md

```diff
@@ -0,0 +1,135 @@
+# Minutes from the Sardana Follow-up Meeting - 2021/03/25
+
+Held on 2021/03/25 at 14:00
+
+Participants: ALBA - Zbigniew Reszela, DESY - Teresa Núñez, MAXIV - Abdullah Amjad, Henrik Enquist, Johan Forsberg, SOLARIS - Michal Piekarski, MBI - Daniel Schick 
+
+## Minutes
+1. Demo of the [h5 session](https://sardana-controls.org/users/scan.html#nxscanh5-filerecorder) concept. 
+    - Issue presented in [this](https://github.com/sardana-org/sardana/pull/1457) PR.
+    - Discussion: find an automatic way of controlling a scan with h5 session (using pre/post scan hooks might be an idea)  
```

Comment:
We also mentioned an idea of a kind of MacroServer startup hook (that would be called automatically after starting the server (an opposite to the atexit). @dschick should we open an issue about it?